core/src/com/unciv/logic/multiplayer/FriendList.kt

```diff
@@ -82,4 +82,13 @@ class FriendList {
         }
         return null
     }
+
+    fun getFriendWithName(name: String): Friend? {
```

Comment:
ByName would be a little more idiomatic - I think? 'with' sounds like you're returning more info, like kotlin .withIndex